README.md

```diff
@@ -1,2 +1,91 @@
-# Tic-Tac-Toe
-This is a tic tac toe game project
+# Tic Tac Toe Game Project
+
+**Two players Tic Tac Toe game.**
+
+## To start
+- Click the green button "Run"
+- Insert player one's name
+- Insert player two's name
+- Player one is X
+- Player two is O
+
+## Roles
+- Player one starts by selecting a valid number from 1 to 9 to place his mark instead of the selected number
+- Player two selects after. Trying to prevent player one to place his marks in a pattern
+
+## To win
+- The player has to place his mark in 3 slots side by side in the following patters:
+
+- (1) If any row
+
+-  X X X
+-  4 5 6
+-  7 8 9
+
+- (2) If any column
+
+-  X 2 3
+-  X 5 6
+-  X 8 9
+
+- (3) If any diagonal
+
+-  X 2 3
+-  4 X 6
+-  7 8 X
+
+- The same thing applies to player two (O)
+
+## Draw state
+- The game draw if the two players marked the 9 slots in the board without one of them completes one of the previous patterns. 
+
+
+## Built With
+
+- Ruby
+
+## Live Demo
+
+[Live Demo Link](https://repl.it/@MahmoudBakr23/Tic-Tac-Toe-Game#main.rb)
+
+## Getting Started
+
+To get a local copy of the repository please run the following commands on your terminal:
+
+$ cd <folder>
+$ git clone
+
```

Comment:
- [ ] Kindly adjust this getting started rules here, it should be `git clone <insert project URL>` then `cd <project folder>`, replace with actual URL and project folder.